### ISSUE TITLE
Valid cron appear as invalid

### ISSUE THREAD

**Issue by xrutayisire** [roccivic/cron-converter#20]
Hello!

First of all, thanks for this library, it helped me a lot to create https://github.com/xrutayisire/react-js-cron.

**Describe the bug**
There are two valid cron sting that are not well process within this library.

Reported in https://github.com/xrutayisire/react-js-cron/issues/30 that is based on this library.

**To Reproduce**

1. First bug
in: `* 2-10,19-23/2 * * *`
actual out: `* 2,4,6,8,10,20,22 * * *`
expected out: `* 2,3,4,5,6,7,8,9,10,19,21,23 * * *`

2. Second bug
in: `* 2-6/2,19-23/2 * * *`
actual out: Error: Invalid value "2-6/2,19-23/2" for hour
expected out: `* 2,4,6,19,21,23 * * *`

**Comment by Sonic0** [comment]
I can confirm the bug because I had the same problem in my Python transposition of this project (https://github.com/Sonic0/cron-converter). I fixed the bug and I probably will be able to make a pull-request with the same changes in JS.

**Comment by roccivic** [comment]
Thanks for the report. Indeed the string parsing looks incorrect.

@Sonic0 How different is your fix to [this commit](https://github.com/roccivic/cron-converter/commit/538fd375ccc5398b22fe2174d3c5ef9107bec5ba)?

**Comment by Sonic0** [comment]
> Thanks for the report. Indeed the string parsing looks incorrect.
> 
> @Sonic0 How different is your fix to [this commit](https://github.com/roccivic/cron-converter/commit/538fd375ccc5398b22fe2174d3c5ef9107bec5ba)?

@roccivic thanks for your work in this library.
I handled the string in the same way as you did. There are 3 minor differences:
1. I included the call to `outOfRange` inside the loop to check every single range and inform the user about the specific wrong part of the string
2. I left the call to `fixSunday` inside the loop, but I think your approach is the better
3. I forgot to add a test for the range as input 😅
